<?xml version="1.0" encoding="utf-8"?>

<launch>

    <node
        name="human_search_node"
        pkg="search"
        type="search_node.py"
        output="screen">
        
        <param name="config_file" value="search_node.yaml" />

    </node>

</launch>
